src/main/steps/upload-your-documents/document-management-client.ts

```diff
@@ -0,0 +1,82 @@
+import Axios, { AxiosInstance, AxiosResponse } from 'axios';
+import FormData from 'form-data';
+
+import type { UserDetails } from '../../app/controller/AppRequest';
+
+export class DocumentManagementClient {
+  client: AxiosInstance;
+  userId: string;
+
+  constructor(options: Options) {
+    this.userId = options.user.id;
+    this.client = Axios.create({
```

Comment:
This doesn't seem as clean as https://github.com/hmcts/nfdiv-frontend/blob/master/src/main/app/case/CaseApi.ts#L14-L18

Reply:
👍 Good idea, I refactored this to take multiple arguments automatically adding them as class properties via the protected method.